app/DynamicsAdapter/DynamicsAdapter.Web/SearchRequest/PersonSearchRequestBuilder.cs

```diff
@@ -0,0 +1,51 @@
+using Fams3Adapter.Dynamics.Identifier;
+using Fams3Adapter.Dynamics.OptionSets.Models;
+using Fams3Adapter.Dynamics.SearchApiRequest;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace DynamicsAdapter.Web.SearchRequest
+{
+    public class PersonSearchRequestBuilder
+    {
+        private PersonSearchRequest _personSearchRequest;
+
+        public PersonSearchRequestBuilder()
+        {
+        }
+
+        public PersonSearchRequestBuilder WithSearchApiRequest(SSG_SearchApiRequest request)
```

Comment:
The builder pattern works the other way around, you want to expose a method that set a property of the object like firstName lastName...., the builder pattern helps with reducing the amount of parameters in the constructor and separate the logic of object initialization
```c#
public PersonSearchRequestBuilder WithFirstName(string firstName) { 
...
}
```
what you are looking for here is to map a SSG_SearchApiRequest to a PersonSearchRequest, a mapping class would suit better, or you can look at [automapper](https://automapper.org/), maybe for latter